recipes/tl/all/conanfile.py

```diff
@@ -0,0 +1,51 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class TlConan(ConanFile):
+    name = "tl"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://tl.tartanllama.xyz"
+    description = "tl is a collection of generic C++ libraries"
+    topics = ("conan", "c++", "utilities")
+    settings = "compiler"
+    license = "CC0-1.0"
+    no_copy_source = True
+    _source_subfolder = "tl"
```

Comment:
```suggestion
    _source_subfolder = "tl"
    
```

Add new empty line, please